pkg/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -126,18 +126,23 @@ func NewNodePerformance(assetsDir string, profile *performancev2.PerformanceProf
 
 		for _, device := range profile.Spec.Net.Devices {
 			devices = make([]string, 0)
-			if device.InterfaceName != nil {
-				devices = append(devices, *device.InterfaceName)
+			if device.DeviceID != nil {
+				devices = append(devices, "^ID_MODEL_ID="+*device.DeviceID)
```

Comment:
The sorting here matters right? Can you please add a comment so nobody will forget in the future?

Reply:
ill add a comment. unittests should ensure this as well